code/game/gamemodes/autotraitor/autotraitor.dm

```diff
@@ -115,35 +115,32 @@
 			traitor_prob += 50
 
 
-		if(traitorcount < max_traitors)
+		if (traitorcount < max_traitors)
 			//message_admins("Number of Traitors is below maximum.  Rolling for new Traitor.")
 			//message_admins("The probability of a new traitor is [traitor_prob]%")
 
-			if(prob(traitor_prob))
-				message_admins("Making a new Traitor.")
-				if(!possible_traitors.len)
-					message_admins("No potential traitors.  Cancelling new traitor.")
-					traitorcheckloop()
-					return
-				var/mob/living/newtraitor = pick(possible_traitors)
-				//message_admins("[newtraitor.real_name] is the new Traitor.")
-
-				forge_traitor_objectives(newtraitor.mind)
-
-				if(istype(newtraitor, /mob/living/silicon))
-					add_law_zero(newtraitor)
+			if (prob(traitor_prob))
+				message_admins("AUTOTRAITOR: making someone traitor")
+
+				if (possible_traitors.len > 0)
+					var/mob/living/traitor_body = pick(possible_traitors)
+
+					if (traitor_body)
+						var/datum/mind/traitor_mind = traitor_body.mind
+
+						if (traitor_mind)
+							log_game("[key_name(traitor_body)] has been auto traitor'ed.")
+							message_admins("AUTOTRAITOR: making [key_name_admin(traitor_body)] a traitor")
+							traitor_body << "<SPAN CLASS='danger'><CENTER><BIG>ATTENTION</BIG></CENTER></SPAN><BR><CENTER>It is time to pay your debt to the [syndicate_name()].</CENTER>"
+							traitor_body = null
+							traitors += traitor_mind
```

Comment:
Others PRs are green now, prolly this change will be just slip into unrelated PRs because it's just a literal cut and paste of lines.